client/components/dropdown-button/style.scss

```diff
@@ -1,6 +1,6 @@
 /** @format */
 
-.woocommerce-dropdown-button {
+.woocommerce_page_wc-admin .woocommerce-dropdown-button {
```

Comment:
If the button is embedded in the activity panel on a page like the orders or coupons page, I'm not sure this rule will work. Does the `woocommerce-page` class added [here](https://github.com/woocommerce/wc-admin/blob/c361948bc3b4c01bfd8e5f38cd3f98a4e77527bd/lib/admin.php#L147) work?